hack/update-test-metadata.sh

```diff
@@ -4,5 +4,17 @@ set -eu
 
 source hack/build-info.sh
 
+readonly METADATA_FILE=".openshift-tests-extension/openshift_payload_cluster-version-operator.json"
+DELETE_EXISTING="${DELETE_EXISTING:-false}"
+
+if [[ "${DELETE_EXISTING,,}" == "true" ]] && [[ -f "${METADATA_FILE}" ]] ; then
+  echo "Removing ${METADATA_FILE}"
+  rm .openshift-tests-extension/openshift_payload_cluster-version-operator.json
```

Comment:
And this file name is dynamic, it was created by https://github.com/openshift/cluster-version-operator/blob/37364492ff31648392a55e22e5086be5a22e86e9/cmd/cluster-version-operator-tests/main.go#L18, so it's better to find a dynamic way to delete it.

@DavidHurta Please correct me if I am wrong.

Reply:
I believe this discussion was resolved in the relevant Slack thread. If not, please ping me.